stdlib/public/stubs/CMakeLists.txt

```diff
@@ -18,8 +18,10 @@ set(LLVM_OPTIONAL_SOURCES
     ${swift_stubs_objc_sources}
     ${swift_stubs_unicode_normalization_sources})
 
-find_package(ICU REQUIRED COMPONENTS uc)
-set(ICU_UC_LIBRARY "")
+if(NOT CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
I'd be all ears for a good solution ;-)
However, the CMAKE_SYSTEM_NAME check is used in several other places as well with similar- ( FindUUID.cmake ) or the same use case ( /stdlib/public/core/CMakeLists.txt ), to name but two examples (there seem to be many more).
If cross-platform compiling is an issue, these things should also be resolved. The code in FindICU.cmake, for example, executes a pkg_check_module() and then simply fills the include and lib information for Android, FreeBSD, Linux and Windows *all with this single pkg_check_module() result set*.
I doubt I'm fluent enough in cmake to assess the work required or solution to be used to make cross compiling foolproof, but possibly the TARGET_TRIPLE variable can be used in cases like the one in stdlib/public/stubs/CMakeLists.txt ?